exercises/kindergarten-garden/introduction.md

```diff
@@ -0,0 +1,6 @@
+# Introduction
+
+The kindergarten class is learning about growing plants.
+The teacher thought it would be a good idea to give them actual seeds, plant them in actual dirt, and grow actual plants.
```

Comment:
Grammar nitpick: apparently the class is planted.

```suggestion
The teacher thought it would be a good idea to have them take actual seeds, plant them in actual dirt, and grow actual plants.
```

Reply:
```suggestion
The teacher thought it would be a good idea to give the class seeds to plant and grow in the dirt.
```